src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -115,6 +118,34 @@ function useSidebarMetadataFetcher(
         [onApiError],
     );
 
+    const updateMetadataInstance = () => {
+        // to be implemented in the next ticket
+    };
+
+    const onCreateSuccessHandler = React.useCallback((templateInstance: MetadataTemplateInstance): void => {
```

Comment:
I don't see much value added with additional declaration of this function. I proposed different solution few lines below